doc/contributing/maintaining-http.md

```diff
@@ -108,16 +108,3 @@ The low-level implementation is made available in the Node.js API through
 JavaScript code in the [lib](https://github.com/nodejs/node/tree/HEAD/lib)
 directory and C++ code in the
 [src](https://github.com/nodejs/node/tree/HEAD/src) directory.
-
-### HTTP2
-
-The low-level implementation of
-[HTTP2](https://nodejs.org/docs/latest/api/http2.html)
-is based on [nghttp2](https://nghttp2.org/). Updates are pulled into Node.js
-under [deps/nghttp2](https://github.com/nodejs/node/tree/HEAD/deps/nghttp2)
```

Comment:
Instead of removing it, I suggest creating a link to the new maintaining document